Controllers/ScoreController.cs

```diff
@@ -498,6 +498,26 @@ private ScoreResponse RemovePositiveModifiers(ScoreResponse s)
             return result;
         }
 
+        [HttpGet("~/scores/modinterface/{hash}/{diff}/{mode}/{context}/{scope}/{method}")]
```

Comment:
Not sure what this endpoint for🤔

Reply:
The Rankvoting is the only big property that is not read by the mod. Therefore, I simply null rankvoting here, so that this is not transmitted unnecessarily.

If you think this is too much work just for that bit of network traffic i can remove it :)

Reply:
It should already be null there as `RankVoting` is not `include`d in query for `v3/scores`

Reply:
Removed this endpoint